app/api/join_game_page.py

```diff
@@ -55,14 +55,14 @@ def join_view():
 
         room = code
 
-        if create != False:
```

Comment:
I think we actually need it set to != False. After looking it up it seems that there's a difference between a boolean 'False' and just falsy values. 
_"However, there might be cases where you specifically want to check if create is True and not just a truthy value. By using if create != False, you are explicitly checking if create is equal to False, ensuring that it's a boolean False value, and not some other falsy value. This can be useful in situations where you expect create to be either True or False and not any other truthy or falsy value."_

Reply:
Woah! good catch! I'm curious exactly what value type request.form is returning? I pushed the revision and turned off PyCharm's reminders for simplified conditionals.

Reply:
Cool! I'll merge now